sdk/iot/Azure.Iot.Hub.Service/src/swagger/iothubservice_modified_readonlyfields.json

```diff
@@ -2450,16 +2450,19 @@
                 },
                 "etag": {
                     "description": "Twin's ETag",
-                    "type": "string"
+                    "type": "string",
+                    "readOnly": true
```

Comment:
Customers need to write to this field when constructing any sort of twin update operation. I can see that Twin Put and some Jobs APIs use this Twin object as an input parameter, so this can't be reaonly

Reply:
Don't we provide the Etag as a parameter to the API; as opposed to setting it in the twin object that you pass into the body?

Reply:
I don't have enough context to tell if this is needed or not. Same with deviceETag. Either way these are recommendations. Make a choice and ultimately only the service team can tell us. Currently we don't even have tests to run things against to verify. My two-cents on this would be to do minimal changes and as we write e2e tests, we might be able to determine these suggestions better.

Reply:
That's a good point; I'd like to keep it as-is, and have the service-team confirm our assumptions and validate with tests/ samples.

Let me know if you'd prefer that I modify a smaller set of values instead?